### PR TITLE
Added class names completion when started typing any string name

### PR DESCRIPTION
**Pull request by mkusher**
Adds #59

### CODE REVIEW

File: src/Padawan/Domain/Completer/ClassNameCompleter.php [hunk 1]
```diff
@@ -27,6 +27,11 @@ public function getEntries(Project $project, Context $context) {
 
     public function canHandle(Project $project, Context $context)
     {
-        return $context->isClassName();
+        return $context->isClassName()
+            || (
+                parent::canHandle($project, $context)
+                && ($context->isString() || $context->isEmpty())
+                && strlen($postfix) > 0
```
Comment (golaod) [inline src/Padawan/Domain/Completer/ClassNameCompleter.php:35]: $postfix doesn't exist

**Comment by golaod** [comment]
You have a bug in your code.

File: src/Padawan/Domain/Completer/ClassNameCompleter.php (hunk 1 above)
Comment (mkusher) [reply src/Padawan/Domain/Completer/ClassNameCompleter.php:35]: yep, that is a problem

**Comment by mkusher** [comment]
@golaod I've fixed the bug and also I've added new test case for classes

**Comment by halftan** [comment]
move to #82

**Pull request closed without merging**